package.json

```
@@ -48,7 +48,7 @@
     "svgo": "1.3.2"
   },
   "devDependencies": {
-    "@commitlint/cli": "9.1.1",
+    "@commitlint/cli": "^9.1.2",
```

This dependency should be pinned again.

```suggestion
    "@commitlint/cli": "9.1.2",
```